docs/contracts.rst

```diff
@@ -341,6 +341,17 @@ Each Contract Factory exposes the following methods.
     Returns the transaction hash for the deploy transaction.
 
 
+.. py:classmethod:: Contract.encodeABI(fn_name, args=None, kwargs=None, data=None)
+
+   Encodes the arguments using the Ethereum ABI for the contract function that
+   matches the given `fn_name` and arguements `args`. The `data` parameter
+   defaults to the function selector
+
+   .. code-block:: python
+
+   >>> Contract.encodeABI(fn_name="register", args=["string",10])
```

Comment:
can you use a string like `"rainbows"` to remove any ambiguity/confusion in case someone thinks that's an ABI type.  Also, space after the `,`.